### PR TITLE
Added functional pie chart

### PR DESCRIPTION
**Pull request by patdbro**
Pie chart allows comparision for three agencie s(dta, humanservices and casa) with small, medium, large and all agencies

image

### CODE REVIEW

File: shiny/observatory/app.R [hunk 1]
```diff
@@ -1,6 +1,88 @@
 library(shiny)
+library(tidyverse)
```
Comment (maxious) [inline shiny/observatory/app.R]: Do we need all of tidyverse or just dplyr?

File: shiny/observatory/app.R [hunk 2]
```diff
@@ -1,6 +1,88 @@
...
+library(readxl)
 library(r2d3)
 
+colName <- "hostname"
```
Comment (maxious) [inline shiny/observatory/app.R]: This code should go at the top of the R file for this tab/visualisation (but keep the library imports in app.R).

File: shiny/observatory/shinytester.R [hunk 3]
```diff
@@ -4,28 +4,19 @@ shinyTester <- tabPanel(
  sidebarLayout(
```
Comment (maxious) [inline shiny/observatory/shinytester.R]: Rename and retitle this file even if it is just "Home"

File: shiny/observatory/app.R [hunk 4]
```diff
@@ -1,6 +1,88 @@
...
+
+if (!exists("data")){
+  data <- read.csv("obs_proto.csv", stringsAsFactors = FALSE)
```
Comment (maxious) [inline shiny/observatory/app.R]: We need a smaller file :)

**Review by maxious — CHANGES_REQUESTED** [review]
This could use some tidying before merging

File: shiny/observatory/app.R (hunk 1 above)
Comment (patdbro) [reply shiny/observatory/app.R]: I use ggplot, stringr, plyr and magrittr which are all in the tidyverse package. Date I will also use lubridate so i find it easier to just use tidyverse

File: shiny/observatory/app.R (hunk 2 above)
Comment (patdbro) [reply shiny/observatory/app.R]: will move to the tab

File: shiny/observatory/shinytester.R (hunk 3 above)
Comment (patdbro) [reply shiny/observatory/shinytester.R]: ok

File: shiny/observatory/app.R (hunk 4 above)
Comment (patdbro) [reply shiny/observatory/app.R]: I know

**Review by maxious — APPROVED** [review]

**Pull request merged**